src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateGlobalUsings_DotNet.cs

```diff
@@ -155,28 +189,8 @@ public void It_can_disable_import_generation()
         {
```

Comment:
Do we need this test at all anymore?  Is it now just testing the same thing as `Implicit_Usings_Are_Not_Enabled_By_Default`?